headlines/src/headlines.rs

```diff
@@ -85,6 +87,22 @@ impl Headlines {
     }
 
     pub fn render_news_cards(&self, ui: &mut eframe::egui::Ui) {
+        if self.config.refresh_news_data{
+                let default_loading_msg = NewsCardData {
```

Comment:
We could do this, but this seems unnecessary. Reason: We're needlessly adding dummy data, only to render a loading state.